### ISSUE TITLE
Add table with trades to general agent monitoring

### ISSUE THREAD

**Issue by gabrielfior** [gnosis/prediction-market-agent#294]
In the dashboard https://deployed-general-agent-viewer.ai.gnosisdev.com/,
we should also add the trades that the agent executed, and a simple KPI with accuracy.

**Comment by vaishnavi-2901** [comment]
can i work on this issue?

**Comment by gabrielfior** [comment]
@vaishnavi-2901 Yes, feel free to pick this up and ask any questions you might have in this issue
